README.md

```diff
@@ -4,7 +4,14 @@
 
 ## Pre-built binaries:
 - Windows / Linux / MacOS: https://nightly.link/AfoninZ/MultiMC5-Cracked/workflows/main/develop
-To update the launcher delete the bin directory and the executable and replace with the newer one. Do not use inbuilt option.
+
+To update the launcher delete the following items and replace with the newer one from the above link. Do not use the inbuilt option, as that downloads the MultiMC launcher instead (this will be fixed in a future version).
+
+- Linux: `bin` directory, `UltimMC` script
+
+- Windows: `iconengines`, `imageformats`, `jars`, `platforms` directories, all files ending with `.dll`, `.dll.a` or `.exe` (turn on file endings).
```

Comment:
Why not just tell the user to replace all the files?

Reply:
Because it might have other folders, like the instances folder or the config file

Reply:
It shouldn't happen anything, because it isn't deleted

Reply:
Eh, if multimc adds a file related to config in the future the readme might need updating, which is annoying

Reply:
What you mean?

Reply:
Oh never mind I thought about your message wrong